src/pages/defaultListTests.tsx

```diff
@@ -21,27 +38,39 @@ export const generateDefaultListTests = ({
         it('can change the visible filters through manage view', () => {})
         it('can remove all filters through manage view', () => {})
     })
+}
 
+export const generateDefaultListRowActionsTests = ({
+    componentName,
+}: {
+    componentName: string
+}) => {
     xdescribe(`${componentName} default row actions tests`, () => {
-        it('should display teh default actions in the actions menu', () => {})
+        it('should display the default actions in the actions menu', () => {})
         it('redirect to the edit page when clicking on the edit action', () => {})
-        it('redirect to the edit page when clicking on the pencil icon', () => {})
-        it('delete an item when pressing teh delete action and confirming', () => {})
-        it('update the list when an item is deleted', () => {})
-        it('show the detail panel when the show details action is clicked', () => {})
-        it('show an edit button in the details panel', () => {})
-        it('be able to copy the api url in the details panel', () => {})
+        it('redirects to the edit page when clicking on the pencil icon', () => {})
+        it('deletes an item when pressing the delete action and confirming', () => {})
+        it('updates the list when an item is deleted', () => {})
+        it('shows the detail panel when the show details action is clicked', () => {})
+        it('shows an edit button in the details panel', () => {})
+        it('can copy the api url in the details panel', () => {})
         it('should open the sharing settings dialog when the sharing settings action is clicked', () => {})
         it('should update the list view when the sharing settings dialog is closed', () => {})
-        it('should open a translation dialog when teh translate action is clicked', () => {})
+        it('should open a translation dialog when the translate action is clicked', () => {})
         it('should successfully save a new translation', () => {})
     })
+}
 
+export const generateDefaultListMultiActionsTests = ({
+    componentName,
+}: {
+    componentName: string
+}) => {
     xdescribe(`${componentName} default multiple actions tests`, () => {
```

Comment:
What's up with these `xdescribe`? 
```suggestion
    describe(`${componentName} default multiple actions tests`, () => {
```